src/main/java/org/isf/opd/gui/OpdBrowser.java

```diff
@@ -101,18 +103,21 @@
  *                   - version is now 1.2.2
  * ------------------------------------------
  */
-public class OpdBrowser extends ModalJFrame implements OpdEdit.SurgeryListener, OpdEditExtended.SurgeryListener {
+public class OpdBrowser extends ModalJFrame implements OpdEdit.SurgeryListener, OpdEditExtended.SurgeryListener, PaginationDataProvider<Opd> {
 
 	private static final long serialVersionUID = 2372745781159245861L;
 
 	private static final SimpleDateFormat dateFormat = new SimpleDateFormat("dd/MM/yy");
-
+	private final int defaultPageSize = 100;
```

Comment:
Why isn't this `final` given it is a constant?   Typically `static final` variables are all upper case to indicate their type.

Reply:
you refer to `dateFormat` or `defaultPageSize`?

Reply:
Final for dateFormat but both should be upper case